dvc/repo/push.py

```diff
@@ -4,7 +4,7 @@
 def push(
     self,
     target=None,
-    jobs=1,
+    jobs=None,
```

Comment:
What about repo/pull.py and others?

Reply:
Didn't notice, but they also has 1 as default. Looks like this was done on purpose. I wonder, which one and whether that still stands.